### ISSUE TITLE
Access and apply panel layout information

### ISSUE THREAD

**Issue by haozhu233** [rstudio/rstudioapi#127]
During the rstudio::conf, @njtierney came up with an great idea to create an addin that can "reset/restore" rstudio panel layout. In the future, it might be nice if we can have two similar functions like `rstudioapi::getThemeInfo()` and `rstudioapi::applyTheme()` that can allow people to access panel info.

**Comment by njtierney** [comment]
Hiya there @jmcphers ! 

I've recently taught a course and was reminded that it would be handy to have this feature, just wondering if this is something that is possible?

**Comment by jmcphers** [comment]
No, it's not possible today. You might be interested in this related issue on the IDE side: https://github.com/rstudio/rstudio/issues/4276
